swarm/storage/localstore.go

```diff
@@ -88,6 +91,10 @@ func NewTestLocalStoreForAddr(params *LocalStoreParams) (*LocalStore, error) {
 func (ls *LocalStore) isValid(chunk Chunk) bool {
```

Comment:
Wouldn't it be better to just run tests with 0 Validators within the `LocalStore`? Why can't we do this?